test/unit-test/FreeRTOS_ARP/FreeRTOS_ARP_utest.c

```diff
@@ -1553,48 +1550,51 @@ void test_eARPGetCacheEntry_IPMatchesOtherBroadcastAddr( void )
     /* =================================================== */
 }
 
-void test_eARPGetCacheEntry_LocalIPIsZero( void )
```

Comment:
We can remove these 2 tests now and re-evaluate if these tests are feasible/valid for endpoint based implementation. If the answer is yes, then let us add them as a part of a new PR.